### PR TITLE
[CIFuzz] Fix MSAN

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fix MSAN in CIFuzz by copying MSAN libraries like we are supposed to.
Fixes https://github.com/google/oss-fuzz/issues/4614

### CODE REVIEW

File: infra/cifuzz/cifuzz.py [hunk 1]
```diff
@@ -427,7 +450,7 @@ def run_fuzzers(  # pylint: disable=too-many-arguments,too-many-locals
     if not testcase or not stacktrace:
       logging.info('Fuzzer %s, finished running.', target.target_name)
     else:
-      utils.binary_print(b'Fuzzer %s, detected error: %s.' %
+      utils.binary_print(b'Fuzzer %s, detected error:\n%s.' %
```
Comment (inferno-chromium) [inline infra/cifuzz/cifuzz.py:453]: nit: remove the "." at end.

File: infra/cifuzz/cifuzz.py [hunk 2]
```diff
@@ -190,6 +195,24 @@ def build_fuzzers(self):
       return False
     return True
 
+  def handle_msan(self):
+    """Copies MSAN libs to |msan_libs_dir| and returns docker arguments to use
+    that directory for MSAN libs."""
+    logging.info('Copying MSAN libs.')
+    os.makedirs(self.MSAN_LIBS_PATH, exist_ok=True)
+    msan_volume_arg = '{msan_libs_path}:{msan_libs_path}'.format(
```
Comment (inferno-chromium) [inline infra/cifuzz/cifuzz.py:203]: nit: nit: f-style var instead of format everywhere.

**Review by inferno-chromium — APPROVED** [review]

File: infra/cifuzz/cifuzz.py (hunk 2 above)
Comment (jonathanmetzman) [reply infra/cifuzz/cifuzz.py:203]: Don't think I can do this. this isn't run using a recent enough python3.

Comment (inferno-chromium) [reply infra/cifuzz/cifuzz.py:203]: fine then, ignore all of these.

**Pull request merged**